.golangci.yml

```diff
@@ -138,14 +170,14 @@ linters-settings:
     sections:
       - standard
       - default
-      - prefix(github.com/tendermint)
+      - prefix(github.com/cometbft)
       - prefix(cosmossdk.io)
       - prefix(github.com/cosmos)
       - prefix(github.com/provenance-io)
       - blank
       - dot
```

Comment:
I've been rethinking these settings. I wonder if it'd be easier on us if we just did

```
      - standard
      - default
      - prefix(github.com/provenance-io)
      - blank
      - dot
```

I don't know about your IDEs, but in IntelliJ, you can't define the auto-import settings to exactly match our current lint settings, so I'm constantly having to fix the imports in files after I make changes. Are our current settings a pain for you too?

Reply:
This hasn't caused too much of an issue for me with VSCode. It could be related to how your IDE is configured.